probes/hasDangerousWorkflowScriptInjection/patch/testdata/userInputAssignedToVariable_fixed.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+on: [pull_request]
+
+env:
+  COMMITS_MESSAGE: ${{ github.event.commits[0].message }}
```

Comment:
The current implementation calls this "COMMIT_MESSAGE". That can lead to a conflict with `head_commit.message`, but I think that's fine for v1...

It's probably a rare conflict, and "COMMITS_MESSAGE" sounds weird, since it's the message of a single commit.